ui/components/app/ledger-instruction-field/ledger-instruction-field.js

```diff
@@ -121,25 +115,14 @@ export default function LedgerInstructionField({ showDataInstruction }) {
     };
   }, [dispatch]);
 
-  const usingLedgerLive = ledgerTransportType === LedgerTransportTypes.live;
   const usingWebHID = ledgerTransportType === LedgerTransportTypes.webhid;
 
-  const isFirefox = getPlatform() === PLATFORM_FIREFOX;
-
   return (
     <div>
       <div className="confirm-detail-row">
         <BannerAlert severity={SEVERITIES.INFO}>
           <div className="ledger-live-dialog">
             {renderInstructionStep(t('ledgerConnectionInstructionHeader'))}
-            {renderInstructionStep(
```

Comment:
If the messages were named like `ledgerConnectionInstructionStep1` etc, this could be a `for` loop.  Would that be better?

Reply:
Yes, we could use it, but currently we only have two instruction steps, and using a for loop might be much for this.